project/AppDependencies.scala

```diff
@@ -5,16 +5,15 @@ private object AppDependencies {
   import play.sbt.PlayImport._
   import play.core.PlayVersion
 
-  private val playPartialsVersion = "6.11.0-play-26"
-  private val httpCachingClientVersion = "9.1.0-play-26"
+  private val playPartialsVersion = "8.1.0-play-26"
+  private val httpCachingClientVersion = "9.5.0-play-26"
   private val playAllowListFilterVersion = "3.4.0-play-26"
   private val validationVersion = "2.1.0"
   private val flexmarkVersion = "0.19.1"
   private val okHttpVersion = "3.9.1"
   private val jsonEncryptionVersion = "4.8.0-play-26"
```

Comment:
Any issues with this version: `4.10.0-play-26`

Reply:
Done